Client/Frontend/Settings/SearchSettingsTableViewController.swift

```diff
@@ -110,7 +110,7 @@ class SearchSettingsTableViewController: ThemedTableViewController {
                 toggle.addTarget(self, action: #selector(didToggleSearchSuggestions), for: .valueChanged)
                 toggle.isOn = model.shouldShowSearchSuggestions
                 cell.editingAccessoryView = toggle
-                cell.selectionStyle = .none
```

Comment:
Had a look and it seems we should make sure to trigger the `onTintColor` to be set in `ThemedSwitch` rather than changing `selectionStyle`. We need to make sure to use the themed color and not the generic Apple blue color. To trigger this, either:
1. we make sure to set `isEnabled = true` on the `toggle`
2. we add the `onTintColor` being set in the `applyTheme` method of that toggle. 

I think option 2 works better to ensure we avoid this problem in the future. Let me know if this doesn't work out

Reply:
I added the **onTintColor** in the **appltyTheme** method of that toggle.